src/Test.ruleset

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<RuleSet Name="Cake Ruleset" ToolsVersion="12.0">
```

Comment:
Perhaps rename to `Cake Bakery Test Ruleset`?

Reply:
Don't know. Kind of like that it says Cake, as it's copy-pasted from the Cake repo 😉